src/ImpactX.cpp

```diff
@@ -97,6 +97,27 @@ namespace impactx
         amrex::ignore_unused(lev);
     }
 
+    /** Resize the mesh, based on the extent of the bunch of particle
+    */
+    void ImpactX::ResizeMesh () {
+        // Extract the min and max of the particle positions
+        amrex::ParticleReal x_min, x_max, y_min, y_max, z_min, z_max;
+        m_particle_container->MinAndMaxPositions(x_min, x_max, y_min, y_max, z_min, z_max);
```

Comment:
Get ready for C++17 nice-ness :heart_eyes: 
```suggestion
        auto const [x_min, x_max, y_min, y_max, z_min, z_max] = m_particle_container->MinAndMaxPositions();
```